src/site/html.rs

```diff
@@ -32,15 +32,17 @@ pub fn build(config: &Config, content: String) -> Result<String> {
         None
     };
 
+    let additional_css = fetch_additional_css(config)?;
+
     let doc: DOMTree<String> = html!(
     <html lang="en" id="oranda" class=theme>
         <head>
             <title>{ text!(&config.name) }</title>
             {homepage}
             {favicon}
             {meta_tags}
-            <link rel="stylesheet" href="https://www.unpkg.com/@axodotdev/fringe/themes/axo-oranda.css"></link>
-            <link rel="stylesheet" href="styles.css"></link>
+            <link rel="stylesheet" href="https://www.unpkg.com/@axodotdev/fringe@0.0.7/themes/axo-oranda.css"></link>
```

Comment:
i thought we were planning on treating fringe the same as the other css and vendoring it?

Reply:
I tried that, we need to do some more stuff since when copying it the fringe file requires other files that are not present

Reply:
oh hrmmm so the fringe file here makes requests to other files?

Reply:
yeah it requires `finge-output` & `theme.css`

Reply:
ok can you open an issue about this and then we can just leave this for now- i may be wrong on this but isn't css that requests other css often a performance issue?